keras_cv/models/backbones/vit/vit_backbone_test.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+import tensorflow as tf
+from absl.testing import parameterized
+from tensorflow import keras
+
+from keras_cv.models.backbones.vit.vit_aliases import ViTTiny16Backbone
+from keras_cv.models.backbones.vit.vit_backbone import ViTBackbone
+
+"""
```

Comment:
I think we can drop this docstring as we're no longer using this testing convention

Reply:
Done!